packages/nuxt/package.json

```
@@ -26,16 +26,16 @@
   "sideEffects": false,
   "exports": {
     ".": {
-      "require": "./dist/index.js",
-      "import": "./dist/index.mjs"
+      "import": "./dist/module.mjs",
+      "require": "./module.cjs"
```

shouldn't this be renamed to `./dist/index.cjs`? that way siroc will generate the file
```suggestion
      "require": "./dist/index.cjs"
```

---

This is a manual file. Rollup's generated cjs format with require syntax (`require('@nuxt/kit')`) is invalid.

---

I see, thanks!